CHANGELOG.md

```diff
@@ -15,6 +15,25 @@ All notable, unreleased changes to this project will be documented in this file.
   - Deprecated the `token` field in order payload, the `id` field should be used
   instead.
 - Enable JWT expiration by default - #9483 by @maarcingebala
+- Add time to dates - #9581 by @IKarbowiak
```

Comment:
Needs to be moved to "Unreleased" above 3.2 version.

Reply:
Is this a breaking change or are we only deprecating fields and adding new ones?

Reply:
Only deprecating and adding new ones.